### PR TITLE
Feral soldiers

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Content "Add new mob: feral soldier"

#### Purpose of change

We need some of mad sodliers

#### Describe the solution

Adding a new mob to feral tree
they have deadly bleed attacks, but has no guns (probably they used it somewhere else and throw it out)

#### Describe alternatives you've considered

not adding it

#### Testing
works fine

### CODE REVIEW

**Comment by NetSysFire** [comment]
If this PR is not yet ready (as indicated by the lorem ipsum), you should mark it as a draft.

**Comment by GuardianDll** [comment]
Sorry, forget about this

File: data/json/monsters/feral_humans.json [hunk 1]
```diff
@@ -645,5 +645,37 @@
     ],
     "anger_triggers": [ "FRIEND_ATTACKED", "PLAYER_CLOSE", "STALK" ],
     "fear_triggers": [ "HURT" ]
+  },
+  {
+    "id": "mon_feral_soldier",
+    "type": "MONSTER",
+    "name": { "str": "feral soldier" },
+    "description": "Lorem ipsum.",
```
Comment (LyleSY) [inline data/json/monsters/feral_humans.json:653]:
```suggestion
    "description": "Wearing a military uniform with a kevlar vest, this person seems to be a highly trained military professional, now turned feral.  Tense, dirty hands grip an automatic rifle as their bloodshot eyes scan for threats.",
```

Comment (GuardianDll) [reply data/json/monsters/feral_humans.json:653]: It uses a knife

File: data/json/monsters/feral_humans.json [hunk 2]
```diff
@@ -645,5 +645,37 @@
...
+    "melee_dice_sides": 2,
+    "melee_cut": 18,
+    "weakpoint_sets": [ "wps_humanoid_body", "wps_humanoid_body_armor", "wps_humanoid_open_helmet" ],
+    "dodge": 4,
+    "harvest": "human",
+    "vision_night": 5,
+    "path_settings": { "max_dist": 40, "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true },
+    "death_drops": "mon_feral_soldier_death_drops",
+    "zombify_into": "mon_zombie_soldier",
+    "fungalize_into": "mon_zombie_fungus",
```
Comment (LyleSY) [inline data/json/monsters/feral_humans.json:677]:
```suggestion
```

Comment (LyleSY) [reply data/json/monsters/feral_humans.json:677]: This entry is used for zombies, not ferals. If you want to create an infected variant to fungalize into like the ones above though, that would be good and cool

Comment (GuardianDll) [reply data/json/monsters/feral_humans.json:677]: Whoops, missed that, thanks

File: data/json/monsters/feral_humans.json (hunk 1 above)
Comment (GuardianDll) [reply data/json/monsters/feral_humans.json:653]: Great, thank you

**Review by LyleSY — APPROVED** [review]

**Comment by LyleSY** [comment]
AFAICT the crafting skill gain test failure isn't related to this monster

**Comment by BioHazardSuit** [comment]
When in doubt, fetch upstream

**Pull request merged**